sema4ai/vscode-client/src/common.ts

```diff
@@ -25,6 +25,10 @@ export type PackageTargetAndNameResult = {
     name: string | null;
 };
 
+export type ActionPackageTargetInfo = PackageTargetAndNameResult & {
+    agentSpecPath: string | null;
```

Comment:
I think I'd just use `agentSpecPath?: string | undefined;` and not pass it in the cases that don't need it (entirely up to you though, feel free to leave as is if you prefer to be explicit everywhere it's constructed).